js/router.js

```diff
@@ -48,6 +48,10 @@ module.exports = Backbone.Router.extend({
         this.navigate(translatedRoute, { trigger: true });
       });
     });
+    
+    history.size = -1;
```

Comment:
> I think it's a REALLY bad idea to muck with a core JS object like History.
Fixed in #1425

> After this PR, you are completely ignoring that existing history and those previous 14 entries in the history queue are ignored and blocked away from the user.
In older versions of application previous history entries were ignored too. Try pressing back key multiple times in v1.1.3.